### ISSUE TITLE
About 2.6.0.1 release

### ISSUE THREAD

**Issue by chenrui333** [savoirfairelinux/opendht#677]
👋 Looks like the latest release build failed, https://github.com/savoirfairelinux/opendht/actions/runs/6132115282/job/16642690850 raise this issue for some awareness. Thanks!

relates to https://github.com/Homebrew/homebrew-core/pull/141874

**Comment by aberaud** [comment]
The code is the same as before.
This might be a Cython bug that breaks backward compatibility.

**Comment by chenrui333** [comment]
yeah, that is fine, I am more concerned about the release semantics 😬 

do you mind marking the release as the latest release? that would be great. thanks!
